plugins/janus_videoroom.c

```diff
@@ -12,7 +12,7 @@
  * different scenarios, ranging from a simple webinar (one speaker, several
  * listeners) to a fully meshed video conference (each peer sending and
  * receiving to and from all the others).
- * 
+ *
```

Comment:
Not sure why you removed all those extra spaces, but ok :smile: 
Did the IDE autofix those for you?